.devcontainer/scripts/notify-dev-entrypoint.sh

```diff
@@ -33,7 +36,7 @@ echo -e "complete -F __start_kubectl k" >> ~/.zshrc
 cd /workspaces/notification-manifests
 
 # kubent
- sh -c "$(curl -sSL https://git.io/install-kubent)"
+sh -c "$(curl -sSL https://git.io/install-kubent)"
 
-# Warm up git index prior to display status in prompt
-git status
```

Comment:
This is unnecessary now as we got rid of starship prompt (it was not playing well with github stale cache).